site-src/concepts/api-overview.md

```diff
@@ -120,6 +120,19 @@ which case the unencrypted byte stream is passed through to the backend.
 You can choose to not terminate TLS, in which case the encrypted byte stream
 is passed through to the backend. 
 
+#### GRPCRoute
+
+!!! info "Experimental Channel"
+
+    The `GRPCRoute` resource described below is currently only included in the
+    "Experimental" channel of Gateway API. For more information on release
+    channels, refer to the [related documentation](https://gateway-api.sigs.k8s.io/concepts/versioning).
+
+GRPCRoute is for idiomatically routing gRPC traffic. Gateways supporting
+GRPCRoute are required to support HTTP/2 without an initial upgrade from HTTP/1,
+so gRPC traffic is guaranteed to flow properly.
+
+
```

Comment:
Nit: Looks like this extra new line isn't necessary

Reply:
Done.